src/python/pants/backend/jvm/tasks/jvm_compile/rsc/rsc_compile.py

```diff
@@ -224,6 +243,7 @@ def _nailgunnable_combined_classpath(self):
     cp = []
     cp.extend(self._rsc_classpath)
     cp.extend(self._scalac_classpath)
+    cp.extend(self._wartremover_classpath)
```

Comment:
Does WartRemover need to be on the compiler classpath? The `-Xplugin:...` flag should be sufficient.